.github/workflows/build-macos.yml

```diff
@@ -39,3 +39,38 @@ jobs:
         with:
           name: node${{ matrix.target-node }}-macos-x64
           path: dist/*
+
+  macos-arm64:
+    runs-on: macos-11.0
+
+    strategy:
+      fail-fast: false
+      matrix:
+        target-node: [14, 16, 18, 19]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest
+
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
```

Comment:
Use matrix.target-node

Reply:
I think that's only the node version that is used for running the build script and doesn't matter as long as its new enough to run it. No need to vary it per target-node.

Reply:
At least to use 18/20 as it's the currently lts

Reply:
Yeah. Probably should update this for all the jobs.